packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -960,6 +969,15 @@ class _InkResponseState extends State<_InkResponseStateWidget>
     }
   }
 
+  void _handleTapUp(TapUpDetails details) {
```

Comment:
Why do we use a separate method? I thought we could simply change `_handleTap` to `_handleTapUp` by adding `TapUpDetails` and let it invoke both the outbounding `onTap` and `onTapUp` at the same time.